04_classwork_basic_calculator/index_calculator.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="calculator.css" type="text/css">
+</head>
+
+<body>
+    <div class="main">
+        <div class="inputForms">
+          <h2>Ultrabasic calculator</h2>
+            <input type="number" id="numberInput" class="numberInput" required="required">
+            <button id="calculateButton" class="calculateButton">Calculate</button>
+            <button class="resetButton" onClick="history.go(0)">Reset</button>
+        </div>
+        <div class="numbers">
+            <ul id="numberlList">
```

Comment:
it's actually a bunch of numbers so maybe **numbersList**?

Reply:
Fixed.